src/test/java/net/masterthought/cucumber/ReportGenerator.java

```diff
@@ -50,14 +49,16 @@ public ReportGenerator() {
         try {
             // points to target/test-classes output
             reportDirectory = new File(ReportGenerator.class.getClassLoader().getResource("").toURI());
+            configuration = new Configuration(reportDirectory, projectName);
+            configuration.setSortingMethod(SortingMethod.ALPHABETICAL);
+            configuration.getEmbeddingDirectory().mkdirs();
```

Comment:
why do we need this mkdir?

is it because of https://github.com/damianszczepanik/cucumber-reporting/pull/889/files#diff-ba17c9b839ba45cadbdc8d1f13f0eea3L118 ?